app.py

```diff
@@ -1,146 +1,123 @@
+    '''
+    if request.method == 'GET':
+        return jsonify()
 
-        return jsonify(data["experience"])
-
-    if request.method == "POST":
+    if request.method == 'POST':
         request_data = request.get_json()
-        experience_data = Experience(**request_data)
-        data["experience"].append(experience_data)
-        index = len(data["experience"]) - 1
-        return jsonify({"id": index})
-
-    if request.method == "DELETE":
-        index = request.args.get("index")
-        if index and index.isdigit():
-            index = int(index)
-            if index in range(len(data["experience"])):
-                removed_entry = data["experience"].pop(index)
-                return (
-                    jsonify(
-                        {
-                            "message": f"Experience entry with index {index} deleted successfully",
-                            "data": removed_entry,
-                        }
-                    ),
-                    200,
-                )
-
-            return jsonify({"error": f"No Experience entry with index {index}"}), 404
-
-        return jsonify({"error": "No index provided"}), 404
+        if test_post_request(request_data):
+            experience_data = Experience(**request_data)
+            data["experience"].append(experience_data)
+            index = len(data["experience"]) - 1
+            return jsonify({"id": index})
 
     return jsonify({})
 
-
-@app.route("/resume/education", methods=["GET", "POST"])
+@app.route('/resume/education', methods=['GET', 'POST'])
 def education():
-    """
+    '''
     Handles education requests
-    """
-    if request.method == "GET":
-        index = request.args.get("index")
+    '''
+    if request.method == 'GET':
+        index = request.args.get('index')
         if index is not None and index.isdigit():
             index = int(index)
             if 0 <= index < len(data["education"]):
                 return jsonify(data["education"][index])
             return jsonify({"error": "Education entry not found"}), 404
         return jsonify(data["education"])
 
-    if request.method == "POST":
+    if request.method == 'POST':
         new_education_data = request.get_json()
-        new_education = Education(**new_education_data)
-        data["education"].append(new_education)
-        new_education_index = len(data["education"]) - 1
-        return jsonify({"id": new_education_index})
+        if test_post_request(new_education_data):
+            new_education = Education(**new_education_data)
+            data["education"].append(new_education)
+            new_education_index = len(data["education"]) - 1
+            return jsonify({"id": new_education_index})
+
     return jsonify({})
 
 
-@app.route("/resume/skill", methods=["GET", "POST", "DELETE"])
+@app.route('/resume/skill', methods=['GET', 'POST', 'DELETE'])
 def skill():
-    """
+    '''
```

Comment:
nit: why are we changing these quotes?

Reply:
I didn't change them. It was updated from another merged PR. I can go ahead and update them.

Reply:
Resolved [here](https://github.com/MLH-Fellowship/orientation-project-python-23.FAL/pull/27/commits/7af1ed34505885c1dff9bf0a630b7c68de392c90)